src/logmessage.cpp

```diff
@@ -122,7 +122,11 @@ namespace g3 {
 
 
    std::string LogMessage::timestamp(const std::string& time_look) const {
-      return  localtime_formatted(_timestamp, time_look);
+      std::ostringstream stringStream;
+
+      stringStream << _timestamp.tv_nsec;
+
+      return g3::localtime_formatted(_timestamp.tv_sec, (time_look.empty() ? g3::internal::date_formatted + " " + g3::internal::time_formatted : time_look)) + "|" + stringStream.str() + "|";
```

Comment:
Please clean this up

1) The checking of empty is basically a work around to avoid having to have a default formatting. Let's put back the default formatting and instead the empty check can go away

2) I don't understand the "I" part. Please explain.